Meta/Lagom/Fuzzers/FuzzCSSParser.cpp

```diff
@@ -16,16 +22,68 @@ struct Globals {
 
 Globals::Globals()
 {
+    // FIXME: Somehow pick a suitable font location and default query
+    Core::ResourceImplementation::install(make<Core::ResourceImplementationFile>("/usr/share/fonts"_string));
+    Gfx::FontDatabase::set_default_font_query("Liberation Sans 10 400 0");
+    Gfx::FontDatabase::the().load_all_fonts_from_uri("resource:///"sv);
```

Comment:
I'm guessing a FontPlugin would solve this problem?

Reply:
This is pretty unfortunate that we need fonts in order to fuzz CSS :thonk:

We could play games like copying the LiberationSerif or SerenitySans font files into the build directory, and load relative to Core::Process::current_executable_path(). But we would need to make sure that the CMakeLists.txt and the BuildFuzzers.sh script and the oss-fuzz Dockerfile (in the oss-fuzz repo) end up copying them into the oss-fuzz test env properly.

Reply:
Loading fonts from the system is also kind of spooky because it makes the tests less reproducible